todoapp/users/admin.py

```
@@ -1,3 +1,18 @@
 
-# Register your models here.
+from users.models import CustomUser
+
+
+class UserAdmin(admin.ModelAdmin):
+    list_display = ("first_name", "last_name", "email", "is_superuser", "is_staff", "last_login")
+    search_fields = ("email", )
+    list_filter = ("date_joined", )
+    readonly_fields = ("last_login",  )
+    
+    def save_model(self, request, obj, form, change):
```

Ruff (0.12.2)

Missing return type annotation for public function `save_model`

Add return type annotation: `None`

(ANN201)